app/javascript/widget/mixins/nextAvailabilityTime.js

```diff
@@ -0,0 +1,256 @@
+import {
+  timeSlotParse,
+  defaultTimeSlot,
+} from 'dashboard/routes/dashboard/settings/inbox/helpers/businessHour.js';
+const { utcToZonedTime } = require('date-fns-tz');
+
+export default {
+  data() {
+    return {
+      dayNames: [
+        'Sunday',
+        'Monday',
+        'Tuesday',
+        'Wednesday',
+        'Thursday',
+        'Friday',
+        'Saturday',
+      ],
+      timeSlots: [...defaultTimeSlot],
+      timeSlot: {},
+    };
+  },
+  computed: {
+    channelConfig() {
+      return window.chatwootWebChannel;
+    },
+    workingHoursEnabled() {
```

Comment:
@iamsivin Do we need this computed property check everywhere since we are checking `workingHoursEnabled` before calling `timeLeftToBackInOnline` mixin?

![CleanShot 2023-05-02 at 20 35 37](https://user-images.githubusercontent.com/12408980/235706917-b9b0c8e1-b237-464b-b757-e1686b3aa755.png)